components/01 - elements/accordions/accordions.twig

```diff
@@ -0,0 +1,29 @@
+<div class="uids-accordion
+  {% if accordion.variant %} uids-accordion--{{ accordion.variant }}{% endif %}"
+  role="tablist"
+  aria-multiselectable="{{ accordion.multiselectable }}"
+>
+  {% set item_number = 1 %}
+  {% for item in accordion.items %}
+    <!-- Use the accurate heading level to maintain the document outline -->
+    <h2 class="uids-accordion__heading">
+      <button class="uids-accordion__button"
+        role="tab" 
+        aria-selected="{{ item.expanded | default('false') }}"
+        aria-expanded="{{ item.expanded | default('false') }}"
+      >
+        {{ item.title }}
+        <i aria-hidden="true" focusable="false" class="fas fa-chevron-up"></i>
+      </button>
+    </h2>
+    {% if accordion.multiselectable == true %}
+      {% set accordion_type = 'multi' %}
+    {% else %}
+      {% set accordion_type = 'single' %}
+    {% endif %}
```

Comment:
Can this be set at the beginning of the template file?

Reply:
Yes, I have done this in my latest round of commits, good suggestion!